reviewer-2-comments.md

```diff
@@ -0,0 +1,13 @@
+# Reviewer 2
+## General comments
+I have reviewed your figure and I respect all the work that was put into it. 
+Incorporating a piece of fruit within a logo is certainly a novel approach.
+That said, I have some major concerns regarding the values that your peach conveys. 
+Thus, I have provided comments to help your peach better symbolize eternal youth and be more accessible for color-blind users.
+
+## Specific comments
```

Comment:
Wwith these specific comments, think about calling out the individual script that needs to be changed. I don't think there's a need for specific lines though, since that technically could vary between groups.